app/javascript/articles/__stories__/podcastArticle.stories.jsx

```diff
@@ -11,7 +11,6 @@ import {
 import '../../../assets/stylesheets/articles.scss';
 
 const ICONS = {
-  REACTIONS_ICON: assetPath('reactions-stack.png'),
```

Comment:
While I was looking through other assets, this one seemed like something that could potentially be from EmojiOne library yet we don't really need it anywhere except these stories so I just removed it. But we should probably update stories at some point as well.